docs/versions.txt

```diff
@@ -13,9 +13,9 @@ JanusGraph.
 [options="header"]
 |==========================
 | JanusGraph | Cassandra | HBase | Bigtable | Elasticsearch | Solr | TinkerPop | Spark | Scala
-| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
-| 0.1.1 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
-| 0.2.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.z*,2.z,5.z |  5.3-5.5.z, 6.2-6.6.z | 3.2.z | 1.6.z | 2.10.z
+| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.7.1, 1.0.0-preZ | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
+| 0.1.1 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.7.1, 1.0.0-preZ | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
```

Comment:
bigtable-hbase simply implements the hbase 1.x api. The internal implementation should not affect consumers of the hbase api.